### PR TITLE
Fix scrolling when adding new tags to the last message in the thread

### LINKED ISSUE #407: Adding new tags to the messages doesn't cause scrolling
This can make it hard to see when adding a new tag that it's happened as it often appears below the bottom of the screen

Before adding 'Does not need reply'
Screenshot 2020-07-29 at 09 08 13

After adding:
Screenshot 2020-07-29 at 09 08 17

### PR DESCRIPTION
**Pull request by ktkization**
Fixes #407

### CODE REVIEW

File: webapp/lib/controller.dart [hunk 1]
```diff
@@ -1300,6 +1300,9 @@ void setMessageTag(model.Tag tag, model.Message message, model.Conversation conv
         view.conversationPanelView
           .messageViewAtIndex(conversation.messages.indexOf(message))
           .addTag(new view.MessageTagView(tag.text, tag.tagId, tagTypeToStyle(tag.type)));
+        if (conversation.messages.indexOf(message) == conversation.messages.length - 1) {
+          view.conversationPanelView.scrollToLatest();
```
Comment (marianamarasoiu) [inline webapp/lib/controller.dart:1304]: can we do the action of scrolling into view directly in the `addTag` method?

File: webapp/lib/view.dart [hunk 2]
```diff
@@ -364,6 +364,10 @@ class ConversationPanelView {
     _conversationWarning.title = '';
     _conversationWarning.classes.add('hidden');
   }
+
+  void scrollToLatest() {
+    _messages.scrollTop = _messages.scrollHeight;
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart:369]: if I remember correctly, there is a `scrollIntoView` method (or something like that) on HTML elements. Could we potentially use that directly on the tag element, if it behave as expected?

File: webapp/lib/controller.dart (hunk 1 above)
Comment (ktkization) [reply webapp/lib/controller.dart]: Changed this.

File: webapp/lib/view.dart (hunk 2 above)
Comment (ktkization) [reply webapp/lib/view.dart:369]: I've tied using ScrollIntoView but it doesn't seem to be working as expected.

**Comment by ktkization** [comment]
Fixed by #433

**Pull request closed without merging**